### ISSUE TITLE
Add "-startdate-enddate" to the filename

### ISSUE THREAD

**Issue by vritme** [ffeast/finam-export#17]
When downloading an array of datasets, separated, for example, by months in one folder - they will be replacing each other in one file with "contract code - timeframe" filename mask.
The only way is to download no more than one query in one folder by one time.
Could filename mask be like "SBRF-TICKS-2020-01-31-2020-12-31"? 
It will automatically solve auto-replacing and losing data therefore, while downloading files in one folder.

Ps. Thanks so much for brilliant lib ;>

**Comment by ffeast** [comment]
@vritme thanks for your suggestion.
It was built like that on purpose so that if you have a trading tool that works with a specific contract one can updates the data, open the tool and see up-to-date values. To support your case I think there could be an additional option indicating how the resulting file must be named. For example, --destfile="{{contract}}-{{timeframe}}-{{start_date}}-{{end_date}}". Does it make sense? Can you submit a PR?

**Comment by vritme** [comment]
Yes, an additional option would be perfect solution, so that everyone could specify the variant they need more.
Will make pull request.

**Comment by vritme** [comment]
PR is done. A bit tricky system with forking -> pr in your own fork -> then go in original project and only then you see your pr and able to link it with original project... But I've made it =).
